### ISSUE TITLE
Remove application_static from docs due to Webpacker changes

### ISSUE THREAD

**Issue by justin808** [shakacode/react-webpack-rails-tutorial#407]
Per this search:

https://github.com/shakacode/react-webpack-rails-tutorial/search?utf8=%E2%9C%93&q=application_static&type=

This needs cleanup.

**Comment by rnice01** [comment]
Hey I'd like to help out with this if I could, seems simple enough.

**Comment by justin808** [comment]
@rnice01 Please start on this! I'll mark this taken by you, but you have only a few days to show progress. Get in touch if you want a Slack invite as well. And we need to update the docs for https://github.com/shakacode/react_on_rails.

**Comment by ramrudra** [comment]
@justin808 I'll pick this up

**Comment by singhvaibhavram** [comment]
@justin808 I would like to take this.

**Comment by justin808** [comment]
@singhvaibhavram Sure. If we can get proper HMR working as well, that would be great.

**Comment by ramrudra** [comment]
@justin808 @singhvaibhavram  PR #446 is for this specific issue.

**Comment by singhvaibhavram** [comment]
Yup. Will try that too. Give me a day to get me acquainted with the project
and then will do that.!!

**Comment by singhvaibhavram** [comment]
@ramrudra can you msg me on slack? I have to ask you something regarding the pull request you made. 
slack email: vaibhavnarayansingh147@gmail.com

**Comment by singhvaibhavram** [comment]
@justin808 I have messaged you on slack for some doubts.

**Comment by ramrudra** [comment]
@singhvaibhavram Sure. 

@justin808 Could you please send me a slack invite on ramrudra.p@gmail.com

**Comment by justin808** [comment]
Closing as consolidated into the broader documentation cleanup in #525. The remaining `application_static` reference is still visible in `docs/jquery-with-react-on-rails.md` and should be handled as part of that current docs pass.
